packages/E2ETest/hasteImpl.js

```diff
@@ -20,6 +20,7 @@ const rnwPath = fs.realpathSync(
 const ROOTS = [rnwPath + path.sep, rnPath + path.sep];
 
 const BLACKLISTED_PATTERNS /*: Array<RegExp> */ = [
+  /msbuild.*/,
```

Comment:
Actually it can be removed.
In the new solution, msbuild -> `react-native bundle` -> msbuild build other parts.
msbuild created a file msbuild.ProjectImports.zip but it can't be accessed by `react-native bundle`
This should be resolved by blacklist in metro.config.js